lib/mirage-solo5.pc

```diff
@@ -6,5 +6,5 @@ Name: mirage-solo5
 Version: 0.1.1
 URL: https://github.com/mirage/mirage-solo5/
 Description: Core platform libraries for Mirage on Solo5
-Libs: ${libdir}/mirage-solo5/libmirage-solo5_bindings.a
+Libs: ${libdir}/mirage-solo5/liboS_stubs.a
```

Comment:
Any particular reason for using a "not fully qualified" library name for the C stubs here, rather than something like `libmirage_solo5_stubs.a`? In general I prefer the latter (i.e. explicit naming) where possible as it tends to be more future proof and robust.